platform/glfw/glfw_gl_backend.cpp

```diff
@@ -0,0 +1,62 @@
+#include "glfw_gl_backend.hpp"
+
+#include <mbgl/gfx/backend_scope.hpp>
+#include <mbgl/gl/renderable_resource.hpp>
+
+#include <GLFW/glfw3.h>
+
+class GLFWGLRenderableResource final : public mbgl::gl::RenderableResource {
+public:
+    GLFWGLRenderableResource(GLFWGLBackend& backend_) : backend(backend_) {
+    }
+
+    void bind() override {
+        backend.setFramebufferBinding(0);
+        backend.setViewport(0, 0, backend.getSize());
+    }
+
+private:
+    GLFWGLBackend& backend;
+};
+
+GLFWGLBackend::GLFWGLBackend(GLFWwindow* window_)
+    : mbgl::gfx::Renderable(
+          [&] {
```

Comment:
why this lamda is capturing `&` ?

Reply:
using `window_`, but I guess that could be passed by value too